src/main/java/frc/robot/Subsystems/OdometryDrivetrain2.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.Subsystems;
+
+import com.spikes2212.command.drivetrains.TankDrivetrain;
+import com.spikes2212.control.FeedForwardSettings;
+import com.spikes2212.control.PIDSettings;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.DifferentialDriveWheelSpeeds;
+import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
+
+/**
+ *This is an abstract class which will be implemented to a drivetrain that will include an odometry,
+ *as well as the methods required to follow a trajectory.
+ *
+ * For a simple example of an implementation, see
+ * <a href=https://docs.wpilib.org/en/stable/docs/software/examples-tutorials/trajectory-tutorial/creating-drive-subsystem.html>here</a>
+ */
+public abstract class OdometryDrivetrain2 extends TankDrivetrain {
```

Comment:
Reformat all documentation in this class according to the documentation conventions